runtime/src/main/cpp/UtilsTest.cpp

```diff
@@ -46,21 +28,21 @@ class PinnedImpl : private kotlin::Pinned {
 } // namespace
 
 TEST(UtilsTest, MoveOnlyImpl) {
-    static_assert(is_nothrow_default_constructible_v<MoveOnlyImpl>, "Must be nothrow default constructible");
-    static_assert(is_nothrow_destructible_v<MoveOnlyImpl>, "Must be nothrow destructible");
-    static_assert(!is_copy_constructible_v<MoveOnlyImpl>, "Must not be copy constructible");
-    static_assert(!is_copy_assignable_v<MoveOnlyImpl>, "Must not be copy assignable");
-    static_assert(is_nothrow_move_constructible_v<MoveOnlyImpl>, "Must be nothrow move constructible");
-    static_assert(is_nothrow_move_assignable_v<MoveOnlyImpl>, "Must be nothrow move assignable");
+    static_assert(kotlin::is_nothrow_default_constructible_v<MoveOnlyImpl>, "Must be nothrow default constructible");
```

Comment:
I'd rather suggest `using namespace kotlin` in the scope of cpp file, for lower noise. When upgraded to C++17 we may be able to keep the main source intact by replacing it with `using namespace std`.
See also the comment at CppSupport.h

Reply:
To be honest, I prefer explicitness (one of the reasons I rather liked going fully qualified names in cpp instead of namespace blocks). And I don't see anything bad in having to replace `kotlin::*` with `std::*` in the future, because it's clear from the code, if we're using our utilities or the standard ones.

However, `using namespace kotlin` is fine IMO, since the entire runtime is supposed to be inside this namespace.
 OTOH `using namespace kotlin::[some_module]` is less fine, because we have many modules and this'd make it harder to understand at a glance where does the symbol come from.
Finally, `using namespace std` is even less fine, at least because of existence of our utilities with the same name as the standard ones, so name collisions are unavoidable.